packages/client/utils/draftjs/convertMobileValueToTaskContent.ts

```diff
@@ -0,0 +1,10 @@
+import {ContentState} from 'draft-js'
+import {convertStateToRaw} from './convertToTaskContent'
+
+const convertAndroidValueToTaskContent = (spacedText: string) => {
+  const text = spacedText.trim()
```

Comment:
Just as you mentioned, the previous implementation deleted spaces within the comment, this also meant it was removing all line breaks entered into the comments, which resulted in non-support of line breaks within the comments. Since this was only happening on Android devices, we decided to solve the problem by only removing the spaces around the comments on Android devices, thereby preserving any line breaks entered by the user within the comments.